### ISSUE TITLE
form parameters' matcher

### ISSUE THREAD

**Issue by musketyr** [cjstehno/ersatz#88]
If form parameters are sent using the POST request it would be great if there is a built-in method on request to match these parameters similar to `query`

```
post('/form') {
    param 'foo', contains(containsString('bar'))
} 
```

**Comment by cjstehno** [comment]
Not a bad idea. I will have to see how much work this would be to implement.

**Comment by musketyr** [comment]
you can use the following decoder snippet to get the parameters out of the body:

```
QueryParameterUtils.parseQueryString(new String(body, 'UTF-8'), 'UTF-8')
```

**Comment by cjstehno** [comment]
This has been implemented and will be in the 1.7.0 release.
